raco/myrial/scanner.py

```diff
@@ -8,7 +8,8 @@
 keywords = ['WHILE', 'DO', 'DEF', 'APPLY', 'CASE', 'WHEN', 'THEN',
             'ELSE', 'END', 'CONST', 'LOAD', 'DUMP', 'CSV', 'SCHEMA',
             'OPP', 'TIPSY', 'UDA', 'TRUE', 'FALSE', 'HASH', 'BROADCAST',
-            'ROUND_ROBIN', 'UNTIL', 'CONVERGENCE', "SYNC", "ASYNC"]
+            'ROUND_ROBIN', 'UNTIL', 'CONVERGENCE', "SYNC", "ASYNC",
+            'ALTER', 'PULL_IDB', 'PULL_EDB', 'BUILD_EDB']
```

Comment:
I'm slightly concerned about the `ALTER` keyword causing misunderstanding (that it means "alter" and not "alternate"). Maybe we should ask some people unfamiliar with this syntax what it suggests to them. An unambiguous *cough* alternative might be `ALTERN` or just `ALTERNATE`.